project/Dependencies.scala

```diff
@@ -85,7 +85,7 @@ object Dependencies {
   val pekkoSerializationJackson = "org.apache.pekko" %% "pekko-serialization-jackson" % pekkoVersion
   val pekkoActorTyped = "org.apache.pekko" %% "pekko-actor-typed" % pekkoVersion
 
-  val logback2 = "net.logstash.logback" % "logstash-logback-encoder" % "4.11"
+  val logback2 = "net.logstash.logback" % "logstash-logback-encoder" % "8.0"
```

Comment:
side note: Why not just call the variable `logstash` instead of `logback2`? But up to you of course.

Reply:
@ioannakok Here is the exclusion I was talking about in the other comment (didn't test if this compiles, but you get the idea :wink:):

```suggestion
  val logback2 = ("net.logstash.logback" % "logstash-logback-encoder" % "8.0").excludeAll(ExclusionRule("com.fasterxml.jackson.core"))
```